beacon.go

```diff
@@ -19,108 +19,95 @@ package beacon
 import (
 	"encoding/xml"
 	"errors"
-	"golang.org/x/oauth2/google"
-	"google.golang.org/api/genomics/v1"
-	"google.golang.org/appengine"
+	"fmt"
 	"net/http"
 	"strconv"
+
+	"cloud.google.com/go/bigquery"
+	"google.golang.org/appengine"
 )
 
 type beaconConfig struct {
-	variantSetIds []string
+	projectID string
+	table     string
 }
 
 var config = beaconConfig{
-	variantSetIds: []string{"3049512673186936334"},
+	projectID: "project-id",
```

Comment:
this should be read from the YAML as environment variables, eg, in app.yaml:

env_variables:
  TABLE: genomics-public-data.platinum_genomes.variants

It's also possible that GOOGLE_CLOUD_PROJECT is already set by the runtime with the project ID

Reply:
done. I didn't find GOOGLE_CLOUD_PROJECT preset so I set it